analysis_options.yaml

```diff
@@ -61,7 +61,7 @@ linter:
     # - avoid_catching_errors # we do this commonly
     - avoid_classes_with_only_static_members
     # - avoid_double_and_int_checks # only useful when targeting JS runtime
-    # - avoid_dynamic_calls # not yet tested
+    - avoid_dynamic_calls # not yet tested
```

Comment:
Remove the comment '# not yet tested' as this is testing it, no?

Reply:
Yes, removed.